### PR TITLE
chore: consolidate adders

### LINKED ISSUE #441: Consider not publishing each adder individually to npm
### Topic

all adders

### Description

Currently we are publishing all adders seperatly to npm (`@svelte-add/tailwindcss`). Additionally they are bundled into the `svelte-add` cli.

Initially this was done to provide an easy to use command to install a single adder. With some of the latest changes we have done lately, this has probably become way more unimportant. Let's compare the two possibilities to apply the tailwindcss adder.
```
pnpm dlx @svelte-add/tailwindcss@latest
```

vs.
```
pnpm dlx svelte-add@latest tailwindcss
```

As of now, both commands contain exactly the same words and have nearly exactly the same number of characters (second is one character shorter). To me the second one feels more natural and would also bring the benefit, that you already know the command for the interactive cli by heart, in case you need it at some point

```
pnpm dlx svelte-add@latest 
```

---

Additionally this could remove the need of the peer dependencies here and solve some of the challenges we had with changesets in the past: https://github.com/svelte-add/svelte-add/blob/b42e0a0169a717b9381843e0689d135a64c3f1f3/packages/cli/package.json#L20-L26

### Describe the solution you'd like

We will need to think of we can properly represent this in the repo, while still keeping some flexibility for #411 . We might want to create a new private package `@svelte-add/adders` that lives in the `./adders` directory. This will work because the adders are bundled into the cli in either way. 

The only thing I'm unsure about is if changesets would still work as we expect in this configuration. We might need to give it a try to find that out.

### PR DESCRIPTION
**Pull request by manuel3108**
Closes #441 

The following things need to be done:

- [ ] deprecate published adder npm packages (i.e. `@svelte-add/tailwindcss`
- [x] Some tests are failing (only one or two per adder)
- [x] Test if it's still possible to create independent community adders (requires modifications, but works)
- [x] Test if the script to update deps is still working as expected
- [x] Test if the cli is working as expected

Notable changes:
- We now have `@svelte-add/adders` instead of many different packages for all adders. This will also allow us to easily implement #411, #445 
- `@svelte-add/adders` is private and will not be published, only bundled by `svelte-add` (and the website)
- I reworked some of the internal dependencies, to remove dependencies from `svelte-add` and rather add them to `@svelte-add/config` where required. I also removed some useless internal dependencies.

### CODE REVIEW

File: packages/testing-library/utils/adder.ts [hunk 1]
```diff
@@ -12,7 +12,11 @@ export async function runAdder(
     adderOptions[adder.config.metadata.id] = optionValues;
     const remoteControlOptions: RemoteControlOptions = { workingDirectory, isTesting: true, adderOptions };
 
-    await executeAdder({ config: adder.config, checks: adder.checks }, remoteControlOptions);
+    await executeAdder(
+        { config: adder.config, checks: adder.checks },
+        { name: "testing-cli", version: "tbd" },
```
Comment (AdrianGonz97) [inline packages/testing-library/utils/adder.ts:17]: is this a leftover?

**Review by AdrianGonz97 — DISMISSED** [review]
everything is looking, as well as working, great!

and this is always a lovely thing to see 😄:
img

File: packages/testing-library/utils/adder.ts (hunk 1 above)
Comment (manuel3108) [reply packages/testing-library/utils/adder.ts]: This generally needs to be there, but I have adjusted the naming to better reflect the package name

**Comment by AdrianGonz97** [comment]
It looks like the newly released version of `pkg-pr-new@0.0.10` is throwing some errors. In `scripts/get-deps-to-publish.js`, can you pin the version by updating `execSync` to:
 ```
execSync(`pnpm dlx pkg-pr-new@0.0.9 publish --pnpm ${paths}`, { stdio: "inherit" });
```

**Comment by manuel3108** [comment]
> It looks like the newly released version of `pkg-pr-new@0.0.10` is throwing some errors. In `scripts/get-deps-to-publish.js`, can you pin the version by updating `execSync` to:
> 
> ```
> execSync(`pnpm dlx pkg-pr-new@0.0.9 publish --pnpm ${paths}`, { stdio: "inherit" });
> ```

Yeah, I have just raised an issue upstream. Will ignore it for now, as we are done this PR and they will probably fix it in a little bit I assume

**Pull request merged**